packages/subcommands/src/register.ts

```diff
@@ -0,0 +1,44 @@
+import { Plugin, postInitialization, postLogin, SapphireClient, Store } from '@sapphire/framework';
+import { SubcommandsPluginOptions, SubcommandsHandler } from './lib/utils/SubcommandsHandler';
+import { join } from 'path';
+import type { ClientOptions } from 'discord.js';
+
+export class Subber extends Plugin {
+	public static [postInitialization](this: SapphireClient, options: ClientOptions): void {
+		Store.injectedContext.subcommandsHandler = new SubcommandsHandler(this, options.subcommands ?? {});
```

Comment:
It would be better to extend the client with the subcommand handler, following conventions from other plugins and also preventing stores that do not need subcommand (exempli gratia event and argument stores) from inheriting the handler.